### ISSUE TITLE
Loss and Prior becomes nan

### ISSUE THREAD

**Issue by austinlaurice** [cemoody/lda2vec#64]
Is this a normal behavior that loss and prior became nan soon after the training process started?
I ran the sample code of 20 news group.

<img width="404" alt="screen shot 2017-12-20 at 5 44 43 pm" src="https://user-images.githubusercontent.com/6941830/34200877-ad584948-e5ad-11e7-932a-4a4beceaa575.png">

**Comment by bountrisv** [comment]
No, I don't think it's working as intended.

**Comment by aggarwalpiush** [comment]
Hi, try to reinstall chainer version 1.9 instead of latest one.

**Comment by richzt** [comment]
changing the code "optimizer.zero_grads()" to "model.cleargrads()",i get a digit.

**Comment by MelvinZang** [comment]
Thank you,  aggarwalpiush's answer is correct. I tried chainer version 1.6.0, and I failed to install it. But 1.9.0 succeeded.

But I have no idea how can I use the latest version of chainer in the model. ttjs8423219's solution is disabled in 3.5.0.

**Comment by ee15btech11014** [comment]
Hi, I'm also getting nan values in loss and prior same as austinlaurice. I even tried changing the chainer versions with 1.9.0 and latest version 7.1.0.
![Screenshot from 2020-01-24 10-56-23](https://user-images.githubusercontent.com/19831552/73046455-31392080-3e98-11ea-91a0-b1238d3fc7c2.png)

Can someone please help me out in resolving the bug.
